src/lib/features/project/project-service.ts

```diff
@@ -383,7 +377,7 @@ export default class ProjectService {
 
         await this.projectStore.create(data);
 
-        const envsToEnable = newProject.environments?.length
+        const envsToEnable = newProject.environments
```

Comment:
This is actually the salient change. The original code relies on length of 0 being falsy.